docs/_theme/djangodocs/static/djangodocs.css

```
@@ -125,8 +125,9 @@ div.versionadded span.title, div.versionchanged span.title, span.versionmodified
 div.versionadded, div.versionchanged, div.deprecated { color:#555; }
 
 /*** p-links ***/
-a.headerlink { color: #c60f0f; font-size: 0.8em; padding: 0 4px 0 4px; text-decoration: none; visibility: hidden; }
-h1:hover > a.headerlink, h2:hover > a.headerlink, h3:hover > a.headerlink, h4:hover > a.headerlink, h5:hover > a.headerlink, h6:hover > a.headerlink, dt:hover > a.headerlink { visibility: visible; }
+a.headerlink { color: #c60f0f; font-size: 0.8em; padding: 0 4px 0 4px; text-decoration: none; opacity: 0; }
```

Please maintain alphabetical ordering of properties:

```suggestion
a.headerlink { color: #c60f0f; font-size: 0.8em; opacity: 0; padding: 0 4px 0 4px; text-decoration: none; }
```

Please can you also change `padding: 0 4px 0 4px;` on `a.headerlink` to `margin-left: 4px;` to remove the spacing inside the focus outline?

```suggestion
a.headerlink { color: #c60f0f; font-size: 0.8em; margin-left: 4px; opacity: 0; text-decoration: none; }
```